exporter/loadbalancingexporter/loadbalancer.go

```diff
@@ -220,6 +220,10 @@ func endpointFound(endpoint string, endpoints []string) bool {
 func (lb *loadBalancer) Shutdown(ctx context.Context) error {
 	err := lb.res.shutdown(ctx)
 	lb.stopped = true
+
+	for _, e := range lb.exporters {
+		err = errors.Join(err, e.Shutdown(ctx))
```

Comment:
maybe this can be optimized down the road with a slice pre-allocated for number of exporters, and append errors to it and call errors.Join at the end. However, this is likely to not matter much here as a few exporters would exist here.

Reply:
Agree.